app/uk/gov/hmrc/agentsexternalstubsfrontend/connectors/AgentsExternalStubsConnector.scala

```diff
@@ -198,4 +199,20 @@ class AgentsExternalStubsConnector @Inject()(
       .recover {
         case Upstream4xxException(e) => throw e
       }
+
+  def createJourneyId(serviceContract: ServiceContract, journeyType: JourneyType)(
+    implicit hc: HeaderCarrier,
+    ec: ExecutionContext): Future[String] = {
+    val json = Json.toJson(serviceContract).as[JsObject] ++ Json.obj("journeyType" -> journeyType)
+    println(s"So $json")
```

Comment:
print line

Reply:
is having a database in the backend for this potentially a little bit of overkill? If the journey ID is only used to get the outcome reason of a particular journey then we could just have a Map of reason -> code like Map("LockedOut" -> "lockedout123", ...). It's only a stub, what do you think?